src/Funds/Funds.js

```diff
@@ -19,23 +20,32 @@ const Funds = () => {
         path={`${FUNDS_ROUTE}/create`}
         render={() => (
           <IfPermission perm="ui-finance.fund-budget.create">
-            <CreateFund />
+            {({ hasPermission }) => (hasPermission
```

Comment:
IMHO, too much repeatable code. It could be something like 
<CheckPermission perm="">
  <CreateFund />
</CheckPermission >